Client/Frontend/Browser/Playlist/BrowserViewController+Playlist.swift

```diff
@@ -13,6 +13,10 @@ private let log = Logger.browserLogger
 
 extension BrowserViewController: PlaylistHelperDelegate {
     
+    private struct DesignUX {
+        static let bravePlaylistOnboardingURL = "https://brave.com/playlist"
```

Comment:
move to BraveUX where we have all urls stored

Reply:
It is moved at https://github.com/brave/brave-ios/pull/4508/commits/9225c3035d7f4104838a1e6fc5b3730824ddeb8e